src/main/java/com/faforever/client/status/Service.java

```diff
@@ -0,0 +1,30 @@
+package com.faforever.client.status;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+@Value
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Service {
```

Comment:
Wouldn't know what but I added a JavaDoc

Reply:
Yeah let me think of what might fit. The javadoc might solve the issue. But just want to avoid people seeing it in the future and thinking it is a meta class for the spring Services when really it is just a dto.